test/test_generated.rb

```diff
@@ -11,14 +11,16 @@ class DataLoadedError < StandardError; end
 
   def generated_without_loading_data(name)
     blob = File.join(samples_path, name)
-    assert_nothing_raised(DataLoadedError, "Data was loaded when calling generated? on #{name}") do
+    begin
       Generated.generated?(blob, lambda { raise DataLoadedError.new })
+    rescue DataLoadedError
+      assert false, "Data was loaded when calling generated? on #{name}"
```

Comment:
I've heard Ryan Davis (author of minitest) rant about this a few times: http://blog.zenspider.com/blog/2012/01/assert_nothing_tested.html